test/integration/cmds/extension/__snapshots__/create.test.js.snap

```diff
@@ -9,7 +9,7 @@ Options:
   -h, --help                 Show help                                 [boolean]
   --id                       Extension id                               [string]
   --name                     Extension name                             [string]
-  --management-token         Contentful management API token            [string]
+  --management-token, --mt   Contentful management API token            [string]
```

Comment:
If I remember correctly for the alias it should be `-mt` with one `-`

Reply:
these are generated by yargs. It seems any multi-letter alias has two dashes and any single letter alias has one?

Reply:
💡